cromwell-helm/templates/cbas-api.yaml

```diff
@@ -28,6 +28,7 @@ data:
       cromwell:
         baseUri: "http://{{ include "app.fullname" . }}-cromwell-svc:8000"
         healthUri: "http://{{ include "app.fullname" . }}-cromwell-svc:8000/engine/v1/status"
+        finalWorkflowLogDir: "https://{{ .Values.persistence.storageAccount }}.blob.core.windows.net/{{ .Values.persistence.blobContainer }}/workspace-services/cbas/{{ .Values.persistence.leoAppInstanceName }}/cromwell-workflow-logs"
```

Comment:
Isn't this the GCP version of the app? If so it probably doesn't want this value

Reply:
I had not realized! Thank makes sense though, I have now removed it